modin/pandas/groupby.py

```diff
@@ -946,6 +946,9 @@ def _wrap_aggregation(
         else:
             groupby_qc = self._query_compiler
 
+        if len(groupby_qc._modin_frame.numeric_columns(True)) == 0:
+            numeric_only = False
```

Comment:
It's better not to access low-level `_modin_frame` from the API layer, it breaks levels independence. Moreover, some of the Modin Frame implementations may not implement `numeric_columns` method, since it isn't part of the Dataframe Algebra (#3661). 

I would suggest using some of the high-level functions here, for example, explicit for-loop over source frame types.

Reply:
fixed